models/global_virtual_group.go

```diff
@@ -6,7 +6,7 @@ import (
 
 type GlobalVirtualGroup struct {
 	ID                    uint64             `gorm:"column:id;primaryKey"`
-	GlobalVirtualGroupId  uint32             `gorm:"column:global_virtual_group_id;index:idx_gvg_id"`
+	GlobalVirtualGroupId  uint32             `gorm:"column:global_virtual_group_id;uniqueIndex:idx_gvg_id_unique"`
```

Comment:
Checked with Alex, for GlobalVirtualGroupId in GlobalVirtualGroup table and GlobalVirtualGroupFamilyId in GlobalVirtualGroupFamily table, they are created as incremental value on chain and would not have any duplicate even if the old entry got removed.

For local virtual group, because when the value of stored_size return to 0, the chain side will delete the local virtual group right away, we decided to not to provide a unique index for lvg, as its logic also does not support one right now. (LocalVirtualGroupId + BucketID is not unique, and can be removed & recreated several times).